code/modules/lighting/lighting_overlay.dm

```diff
@@ -3,7 +3,6 @@
 	mouse_opacity = 0
 	simulated = 0
 	anchored = 1
-	flags = NOREACT
```

Comment:
The bomb so colossal that its detonation consumes the very fabric of space, time, light, etc.

Transcended Pete